### ISSUE TITLE
Defect-level-2: Register for care, address of facility is missing

### ISSUE THREAD

**Issue by AngelaFowler82** [department-of-veterans-affairs/va.gov-team#22311]
Label: 18928

    - [508-defect-2]
     
https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2

Issue Title: Defect level 2, Register for care, address of facility is missing.
Point of Contact: Angela Fowler
**VFS Point of Contact:** _First name only: Angela

<h2>User Story or Problem Statement:</h2>
As a user with a disability who relies on public transportation, I need the exact address of a destination when booking Uber or Para-Transit, or planning a rout on the city bus.

<h2>Details</h2>
Under Register for Care, Patient Registration Admissions, the address of the facility is missing. While there is a link to the map, locating the address on the map page greatly increases cognative load.

<h2>Acceptance Criteria</h2>

- [ ] The facility address has been added to the Register for Care page, under Patient Registration Admissions.

Environment

Steps to Recreate

Proposed Solution (if known)

WCAG or Vendor Guidance (optional)

Screenshots or Trace Logs

**Comment by laflannery** [comment]
This would be solved if the Top Task page was used rather than the VAMC Detail page. The top task page is currently a draft on prod

**Comment by laflannery** [comment]
Top Task page has been published
